### PR TITLE
[vividus-plugin-web-app] Add ability to use visibility in nested steps

### PR DESCRIPTION
**Pull request by draker94**

### CODE REVIEW

File: docs/modules/plugins/pages/plugin-web-app.adoc [hunk 1]
```diff
@@ -760,3 +760,32 @@ Then `${info.duration}` is > `1000`
 Then `${info.networkState}` is = `2`
 Then `${info.src}` matches `.+youtube.+`
 ----
+
+=== Perform steps for all elements
+
+Executes the steps against all elements found by locator.
```
Comment (valfirst) [inline docs/modules/plugins/pages/plugin-web-app.adoc:766]: we need to define explicitly, that the current iteration element is set as a search context, and describe how context is managed by the step

File: vividus-tests/src/main/resources/story/integration/GenericNestedSteps.story [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Description: Integration tests for GenericNestedSteps class.
```
Comment (valfirst) [inline vividus-tests/src/main/resources/story/integration/GenericNestedSteps.story:1]:
```suggestion
```

File: vividus-tests/src/main/resources/story/integration/NestedSteps.story [hunk 3]
```diff
@@ -1,14 +1,9 @@
...
-    @requirementId 2054
-
-Given I am on a page with the URL '${vividus-test-site-url}/elementState.html'
-When I find > `0` elements `id(element-to-hide)` and while they exist do up to 10 iteration of
-|step                                             |
-|When I click on element located `id(button-hide)`|
+    @epic vividus-plugin-web-app
+
+Scenario: Verify step "I find $comparisonRule `$number` elements by `$locator` and for each element do$stepsToExecute"
+Given I am on a page with the URL 'https://demo.guru99.com/'
```
Comment (valfirst) [inline vividus-tests/src/main/resources/story/integration/NestedSteps.story:5]: we're changing our tests to do not depend on external resources, if our test site doesn't have needed functionality, please feel free to open a PR

**Review by valfirst — CHANGES_REQUESTED** [review]

**Review by valfirst — APPROVED** [review]

File: docs/modules/plugins/pages/plugin-web-app.adoc [hunk 4]
```diff
@@ -760,3 +760,36 @@ Then `${info.duration}` is > `1000`
 Then `${info.networkState}` is = `2`
 Then `${info.src}` matches `.+youtube.+`
 ----
+
+=== Perform steps for all elements
```
Comment (ikalinin1) [inline docs/modules/plugins/pages/plugin-web-app.adoc:764]: It's more like for-each not for all?

**Review by uarlouski — APPROVED** [review]

**Review by ikalinin1 — APPROVED** [review]

**Pull request merged**